core/blockly.js

```diff
@@ -175,14 +175,19 @@ Blockly.svgResize = function(workspace) {
 };
 
 /**
- * Handle a key-down on SVG drawing surface.
+ * Handle a key-down on SVG drawing surface. Does nothing if the main workspace is not visible.
  * @param {!Event} e Key down event.
  * @private
  */
+ // TODO handle cases where there are multiple workspaces and non-main workspaces are able to accept
```

Comment:
From the style guide: "**DO** Create GitHub issues with TODOs and link them using TODO(#issueNumber)."

The issue should be on Blockly since it's an upstream problem.

Reply:
done